addons/website_blog/static/src/js/website_blog.js

```diff
@@ -68,19 +73,19 @@ publicWidget.registry.websiteBlog = publicWidget.Widget.extend({
      */
     _onShareArticle: function (ev) {
         ev.preventDefault();
-        var url = '';
-        var $element = $(ev.currentTarget);
-        var blogPostTitle = $('#o_wblog_post_name').html() || '';
-        var articleURL = window.location.href;
-        if ($element.hasClass('o_twitter')) {
+        let url = "";
+        const currentTargetEl = ev.currentTarget;
+        const blogPostTitle = document.querySelector("#o_wblog_post_name").innerHTML || "";
```

Comment:
Why did we even take the `html` if that's just supposed to be text? Let's use `textContent`.